Tests/BootCamp.Chapter.Tests/DeckTest.cs

```diff
@@ -48,9 +48,7 @@ public void DrawAt_OutOfIndex_Throws_IndexOutOfRangeException(int index)
         [Fact]
         public void DrawRandom_52Times_Returns_SameCards_DifferentOrder()
         {
-            List<Card> cards = BuildFullListOfCards();
-            IDeck deck = new Gambling.Deck(cards);
-            List<Card> newCards = new List<Card>();
+            BuildListOfCardsGiveItToDeckAndCreateEmptyListOfCards(out List<Card>  cards, out IDeck deck, out List<Card> newCards);
```

Comment:
Too long just use a theory and takes out List<Card>  cards, out IDeck deck, out List<Card> newCards as input in test.